src/test/java/io/stargate/sgv2/jsonapi/service/shredding/ShredderDocLimitsTest.java

```diff
@@ -48,7 +48,7 @@ public void catchTooBigDoc() {
           .isInstanceOf(JsonApiException.class)
```

Comment:
@tatu-at-datastax see this, this test is still green, although I completely changed the properties.. usually when writing tests that depend on the config you should assert that the changes in the config are applied in the tests and would break it.. For example, by explicitly setting values in the test profile, like here:

```
@QuarkusTest
@TestProfile(InsertManyCommandTest.Profile.class)
class InsertManyCommandTest {

  public static class Profile implements NoGlobalResourcesTestProfile {

    @Override
    public Map<String, String> getConfigOverrides() {
      return Map.of("stargate.jsonapi.doc-limits.max-insert-many-documents", "2");
    }
  }
```

The `InsertManyCommandTest` fails now, signaling that there was a breaking change.

Reply:
@ivansenic I think this is @maheshrajamani 's limit; I worked only on document limits. I agree that testing of breaking limits (and yes included changes to defaults) should be tested.

Reply:
Resolving.